src/ResourceManager/Cdn/Commands.Cdn/Profile/GetAzureRmCdnProfile.cs

```diff
@@ -19,26 +19,29 @@
 using Microsoft.Azure.Commands.Cdn.Models.Profile;
 using Microsoft.Azure.Commands.Cdn.Properties;
 using Microsoft.Azure.Management.Cdn;
+using System;
+using System.Net;
+using Microsoft.Azure.Management.Cdn.Models;
 
 namespace Microsoft.Azure.Commands.Cdn.Profile
 {
     /// <summary>
     /// Defines the New-AzureRMCdnProfile cmdlet.
     /// </summary>
-    [Cmdlet(VerbsCommon.Get, "AzureRMCdnProfile"), OutputType(typeof(PSProfile))]
+    [Cmdlet(VerbsCommon.Get, "AzureRmCdnProfile"), OutputType(typeof(PSProfile))]
     public class GetAzureRmCdnProfile : AzureCdnCmdletBase
     {
         /// <summary>
         /// Gets or sets the profile name.
         /// </summary>
-        [Parameter(Mandatory = false, HelpMessage = "Azure Cdn profile name.")]
+        [Parameter(Mandatory = false, HelpMessage = "Azure CDN profile name.")]
         [ValidateNotNullOrEmpty]
         public string ProfileName { get; set; }
 
         /// <summary>
         /// The resource group name of the profile.
         /// </summary>
-        [Parameter(Mandatory = false, HelpMessage = "The resource group of the Azure Cdn Profile")]
+        [Parameter(Mandatory = false, HelpMessage = "The resource group of the Azure CDN profile.")]
```

Comment:
ValueFromPipelineByPropertyName allows piping from generic cmdlets.  Also, please add alias to 'ResourceGroup'